actionpack/lib/action_dispatch/routing/redirection.rb

```diff
@@ -137,6 +137,9 @@ module Redirection
       #
       #   get "/stories" => redirect("/posts")
       #
+      # This causes redirection ignoring parts of the URI, query string, etc.
```

Comment:
`This causes redirection` --> `This will redirect the user, while ignoring certain parts of the request, including...`